lib/gis/parser_interface.c

```diff
@@ -107,7 +107,7 @@ void G__usage_xml(void)
     char *type;
     char *s, *top;
     int i;
-    const char *encoding = NULL;
+    const char *encoding = "";
```

Comment:
Which is better?

If the `NULL` is better:
```suggestion
    const char *encoding = NULL;
```